sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
-
+*  Fixed retrying write operations when a gone exception occurs. - See [PR 35838](https://github.com/Azure/azure-sdk-for-java/pull/35838)
```

Comment:
Since this is a behavior breaking change, should we mention it in other changes section with some more details on why we are making this change and what it means for the customers?

Reply:
Made the change.